### PR TITLE
Fix typos in comments and test code

### PR DESCRIPTION
**Pull request by 304NotModified**

### CODE REVIEW

File: src/NLog/Common/InternalLogger.cs [hunk 1]
```diff
@@ -452,7 +452,7 @@ private static void WriteToErrorConsole(string message)
         /// <param name="message">A message to write.</param>
         /// <remarks>
         /// Works when property <see cref="LogToTrace"/> set to true.
-        /// The <see cref="System.Diagnostics.Trace"/> is used in Debug and Relese configuration. 
+        /// The <see cref="System.Diagnostics.Trace"/> is used in Debug and Relaese configuration.
```
Comment (snakefoot) [inline src/NLog/Common/InternalLogger.cs]: Relaese -> Release ?

Comment (304NotModified) [reply src/NLog/Common/InternalLogger.cs]: Lol thanks!

Comment (304NotModified) [reply src/NLog/Common/InternalLogger.cs]: FYI, Mostly used Resharper for this, but it was Al repeating task (one spell error, 1 keycombo + enter)

Comment (304NotModified) [reply src/NLog/Common/InternalLogger.cs]: fixed in 6c5cee0b2df841253c2f71507864bf4e5bce8ee7

**Pull request merged**